cloud/disk_manager/internal/pkg/dataplane/url/vhd/common.go

```diff
@@ -0,0 +1,29 @@
+package vhd
+
+import "github.com/ydb-platform/nbs/cloud/disk_manager/internal/pkg/dataplane/url/common"
+
+////////////////////////////////////////////////////////////////////////////////
+
+type imageMapEntry struct {
+	data   bool
```

Comment:
It should be hasData. data bool just sounds not right.

Reply:
no, this field has the same naming with imageMapEntry from qemu-img map 
https://github.com/ydb-platform/nbs/blob/bf37ab3e6fb00f5093ccd4d589f9875edd0e0abf/cloud/disk_manager/internal/pkg/dataplane/url/common/image_map.go#L19